app/presenters/base_registration_presenter.rb

```diff
@@ -21,6 +21,10 @@ def display_convictions_check_message
     end
   end
 
+  def in_progress?
+    # TODO: For now all registrations are submitted in the new system. To update when we build front end flow.
+    false
+  end
```

Comment:
```suggestion
  end
 
```

Looks to be missing a line between the methods

Reply:
🙇‍♀